src/Providers/LighthouseServiceProvider.php

```diff
@@ -45,7 +45,7 @@ public function boot(): void
         $this->mergeConfigFrom(__DIR__.'/../../config/config.php', 'lighthouse');
 
         $this->publishes([
-            __DIR__.'/../../config/config.php' => config_path('lighthouse.php'),
+            __DIR__.'/../../config/config.php' => base_path('config/lighthouse.php'),
```

Comment:
This is not the same. 
I can move my `config` folder another place if I want to.
You should do something like this https://github.com/nuwave/lighthouse/pull/645/files#diff-59794b479ab0add30d46c8336d474322R54 instead